hook/DestIPFoundHook.js

```diff
@@ -357,30 +362,24 @@ class DestIPFoundHook extends Hook {
         return; // reserved black hole and blue hole...
       }
       
-      this.appendNewFlow(ip,fd);
+      this.appendNewFlow(ip, fd);
     });
 
-    sem.on('DestIP', (event) => {
```

Comment:
don't delete this, this is a debugging interface to manually trigger destip action by redis publish